packages/backend-plugin-api/api-report.md

```diff
@@ -333,11 +333,19 @@ export interface HttpAuthService {
   }>;
 }
 
+// @public (undocumented)
+export interface HttpRouterHealthCheckConfig {
+  // (undocumented)
+  handler: () => Promise<any>;
+}
+
 // @public (undocumented)
 export interface HttpRouterService {
   // (undocumented)
   addAuthPolicy(policy: HttpRouterServiceAuthPolicy): void;
   // (undocumented)
+  healthCheckConfig(healthCheckOptions: HttpRouterHealthCheckConfig): void;
```

Comment:
Not sure this should live on the http router tbh. Or framed a bit differently: if we can make this a separate service we probably should. I'm thinking this should probably be additive as well so that it's possible to add health checks from modules etc.